macOS/DuckDuckGo/Preferences/Model/PreferencesSidebarModel.swift

```diff
@@ -309,26 +309,19 @@ final class PreferencesSidebarModel: ObservableObject {
     }
 
     private func makeSubscriptionState() async -> PreferencesSidebarSubscriptionState {
-        let currentSubscriptionFeatures = await subscriptionManager.currentSubscriptionFeatures()
+        // This requires follow-up work:
+        // https://app.asana.com/1/137249556945/task/1210799126744217
+        let currentSubscriptionFeatures = (try? await subscriptionManager.currentSubscriptionFeatures()) ?? []
         let shouldHideSubscriptionPurchase = subscriptionManager.currentEnvironment.purchasePlatform == .appStore && subscriptionManager.canPurchase == false
 
         if subscriptionManager.isUserAuthenticated {
             // Calculate current user entitlements
             var currentUserEntitlements: [SubscriptionEntitlement] = []
             let entitlements: [SubscriptionEntitlement] = [.networkProtection, .dataBrokerProtection, .identityTheftRestoration, .identityTheftRestorationGlobal, .paidAIChat]
 
-            if let subscriptionManagerV2 = subscriptionManager as? SubscriptionManagerV2,
-               let tokenContainer = try? await subscriptionManagerV2.getTokenContainer(policy: .localValid) {
-
-                for entitlement in entitlements where tokenContainer.decodedAccessToken.hasEntitlement(entitlement) {
+            for entitlement in entitlements {
+                if let hasEntitlement = try? await subscriptionManager.isFeatureEnabled(entitlement.product), hasEntitlement == true {
                     currentUserEntitlements.append(entitlement)
-
-                }
-            } else {
-                for entitlement in entitlements {
-                    if let hasEntitlement = try? await subscriptionManager.isEnabled(feature: entitlement.product), hasEntitlement == true {
-                        currentUserEntitlements.append(entitlement)
-                    }
```

Comment:
Diff looks awful here, I'm just standardizing the way to check if the feature should be greyed out between v1 and v2.